remoteappmanager/handlers/base_handler.py

```diff
@@ -8,7 +8,7 @@ class BaseHandler(web.RequestHandler, LoggingMixin):
     """Base class for the request handler."""
 
     def get_current_user(self):
-        return self.settings["user"]
+        return self.application.user.name
```

Comment:
This will return a User object in the future, but for now, we just hand out the string as before.